Src/FluentAssertions/TypeDescriptionUtility.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace FluentAssertions
+{
+    internal static class TypeDescriptionUtility
```

Comment:
🔧 Please keep this in the `Data` folder. It's currently only used by the code in that folder and should be treated as an implementation detail. It also seems to overlap with what `IValueFormatter` is doing, but it seems to be more specific for your particular use case.

Reply:
I don't see an `IValueFormatter` implementation for `Type` values. This utility class really is specific to this use case.

Reply:
Exactly. That's why I want it to be local to the implementation. We do apply DRY, but only within boundaries.

Reply:
Okay, but all of the places that call into this type are in the root, and none of them are in `Data`. Should `TypeDescriptionUtility` still be in `Data`, given that _nothing in `Data` uses it_??

Reply:
Ah yes, you're right. My bad. In that case, move it to `Common`

Reply:
Done. :-)